lib/nsxv.py

```diff
@@ -326,14 +329,25 @@ def __str__(self):
     for s in services:
       service = '%s%s' % (service, s)
 
+    #applied To
```

Comment:
Is this a comment or something else? There should be a space after the # if this is a comment and the capitalization is incorrect for a sentence. Please fix.